src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ConstantNameCheck.java

```diff
@@ -61,10 +61,23 @@
  * <p>
  * To configure the check:
  * </p>
+ * <p>
+ * The following configuration is applicable by default.
+ * </p>
  * <pre>
  * &lt;module name="ConstantName"/&gt;
  * </pre>
- *
+ * <p>Example:</p>
+ * <pre>
+ * class MyClass {
+ *   public final static int FIRST_CONSTANT1 = 10; // OK
+ *   protected final static int SECOND_CONSTANT2 = 100; // OK
+ *   final static int third_Constant3 = 1000; // violation, name 'third_Constant3' must
+ *                                           // match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'
+ *   private final static int fourth_Constant4 = 50;// violation, name 'fourth_Constant4' must match
```

Comment:
`private final static int fourth_Constant4 = 50;//`
Please add space here ------------------------------^

Reply:
I added but it showed that limit of characters exceeded(100), mine was 101 so I used a backspace

Reply:
I had added them in the beginning and it showed an error that line length exceeded(100 is max) mine was 101, so I used a backspace. I rechecked it but the error was still there.
[ERROR] [checkstyle] [ERROR] E:\checkstyle\src\main\java\com\puppycrawl\tools\checkstyle\checks\naming\ConstantNameCheck.java:74: Line is longer than 100 characters (found 101). [LineLength]

Reply:
Please change constant name to other name, which is shorter, to satisfy line length check

Reply:
done.